vsn:

    goal: 0
    camera_topic: "/camera/color/image_raw"
    pirlnav_path: '/home/gram/rafarepos/ros4vsn/examples/semnav'
    show_screen_images: False
